utils/filters.ts

```diff
@@ -14,14 +14,13 @@ export const toPercent = (value: number | undefined | BN): string => {
   }
 
   if (isNumber(value)) {
-    return Number.isInteger(value) ? `${value}.00 %` : `${value} %`
+    const floatNum = Number.parseFloat(value.toString()).toFixed(2)
+    return `${floatNum} %`
   }
 
-  if (BN.isBN(value)) {
```

Comment:
**Note**: some logic changed here.

`value` type has explicitly declared as `number`, `undefined` and `BN`, we don't need to check again.

here, it's type is `BN` only.

Reply:
Maybe @vikiival can take a peak 😇

Reply:
this filter is not used for now

Reply:
> type has explicitly declared

It could still fail in runtime.

> this filter is not used for now

I would keep it for future royalties component

Reply:
@riton @vikiival 
kept it!

Reply:
@vikiival @roiLeo 
kept the filter finally.